lab3/task2.js

```diff
@@ -0,0 +1,19 @@
+let triangle = document.getElementsByClassName("triangle")[0];
+let p = document.createElement('p');
+triangle.appendChild(p);
+let arr = [50];
+let i, j;
+let size = 25;
+for (i = 1; i <= size; i++) {
```

Comment:
You can write this in another way
`for (let i = 1; i <= size; i++)`
Please rewrite like this and delete `let i, j;` before